src/Transformer/NormalizationTransformer.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flagbit\Bundle\CategoryBundle\Transformer;
+
+/**
+ * Interface for normalization category property data transformers.
+ */
+interface NormalizationTransformer
+{
+    /**
+     * Apply transformations to the supplied data on import.
+     *
+     * @param mixed $data a single property value
+     *
+     * @return mixed
```

Comment:
The output would be string in all cases. Strings are ending up in the exported files.